nuget/Microsoft.Windows.CsWinRT.Authoring.targets

```diff
@@ -52,6 +53,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <PropertyGroup>
       <CsWinRTAuthoringInputs>$(CsWinRTAuthoringInputs) @(CsWinRTAuthoringDistinctWinMDs->'"%(FullPath)"', ' ') </CsWinRTAuthoringInputs>
     </PropertyGroup>
+
+    <!--
+      Make sure 'CsWinRTExeTFM' is set in authoring scenarios as well. The items in this
+      group should be kept in sync with the ones in 'Microsoft.Windows.CsWinRT.targets'.
+    -->
+    <PropertyGroup>
+      <CsWinRTExeTFM Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) == 5">net5.0</CsWinRTExeTFM>

```

Comment:
CsWinRT doesn't support `net5.0` and `net8.0` as targets ATM, wouldn't this lead to a failure?

Reply:
I don't disagree, but let's remove .NET 5 from everywhere in a separate follow up PR 👍